lib/matplotlib/_mathtext.py

```diff
@@ -2599,3 +2603,30 @@ def auto_delim(self, s, loc, toks):
             # if "mid" in toks ... can be removed when requiring pyparsing 3.
             toks["mid"].asList() if "mid" in toks else [],
             toks["right"])
+
+    def boldsymbol(self, s, loc, toks):
+        self.push_state()
+        state = self.get_state()
+        lw = list(map(chr, range(ord('a'), ord('z') + 1)))
+        up = list(map(chr, range(ord('A'), ord('Z') + 1)))
+        smGreek = list(map(lambda a: unicodedata.name(chr(a))
+                           .split()[-1]
+                           .lower(),
+                           range(ord('\N{GREEK SMALL LETTER ALPHA}'),
+                                 ord('\N{GREEK SMALL LETTER OMEGA}') + 1)))
```

Comment:
Should rename for PEP8, and maybe also prefer list comprehensions
```suggestion
        lower_greek = [
            unicodedata.name(chr(a)).split()[-1].lower()
            for a in range(ord('\N{GREEK SMALL LETTER ALPHA}'),
                           ord('\N{GREEK SMALL LETTER OMEGA}') + 1))]
```